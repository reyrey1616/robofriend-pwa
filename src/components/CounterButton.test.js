import { shallow } from 'enzyme';
import React from 'react';
import CounterButton from './CounterButton';

it('Renders the Counter Button Component', () => {
  expect(shallow(<CounterButton />)).toMatchSnapshot();
});
