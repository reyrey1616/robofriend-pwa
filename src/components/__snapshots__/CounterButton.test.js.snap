// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Counter Button Component 1`] = `ShallowWrapper {}`;
